webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/RuleGrid.scala

```diff
@@ -231,14 +233,16 @@ class RuleGrid(
               refreshTable("${htmlId_rulesGridId}", ${newData.json.toJsCmd});
               ${futureCompliance.toJsCmd}
               ${futureChanges.toJsCmd}
-          """)
+          """) // JsRaw ok, escaped
           }) match {
             case Full(cmd) =>
               cmd
             case eb: EmptyBox =>
               val fail = eb ?~! ("an error occured during data update")
               logger.error(s"Could not refresh Rule table data cause is: ${fail.msg}")
-              JsRaw(s"""$$("#ruleTableError").text("Could not refresh Rule table data cause is: ${fail.msg}");""")
+              JsRaw(
+                s"""$$("#ruleTableError").text("Could not refresh Rule table data cause is: ${fail.msg}");"""
+              ) // JsRaw ok, no user inputs
```

Comment:
I really hope our error messages are not "injectable", but I'm still concerned about this line.. I'd be relieved if the failure message was escaped :grin: